docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,33 @@
 ---
 title: Cloud
 description: Teleport as a service
-h1: Teleport cloud
+h1: Teleport Cloud
 videoBanner: 1jhKOtBinm4
 ---
 
-We run Teleport Cloud as hosted, managed Teleport as a service.
-Connect your nodes, web applications, kubernetes clusters and databases.
+# Teleport Cloud
+
+Teleport Cloud provides a SOC 2 certified Highly Available (HA) hosted Teleport Cluster service
```

Comment:
It might be clearer to start with the fundamentals, then add advantages later in the paragraph. Maybe something like,

```markdown
Teleport Cloud is a hosted Teleport cluster that we provide as a service. 
The Teleport Auth Service and Proxy Service are managed by Teleport and 
deployed in a secure tenant that is dedicated to your organization. Teleport 
Cloud is SOC 2 certified and highly available, and Teleport handles scaling, 
patching, monitoring, and upgrading the Auth Service and Proxy Service.
```